zipline/algorithm.py

```diff
@@ -424,6 +424,9 @@ def _create_generator(self, sim_params, source_filter=None):
         processed by the zipline, and False for those that should be
         skipped.
         """
+        if sim_params is not None:
```

Comment:
this wasn't necessary for any tests, but seemed odd not to have.  we pass in sim_params into this method and they weren't being used.